app/components/using_these_materials_component.rb

```diff
@@ -12,4 +12,12 @@ def initialize(document:)
   def repository_url
     document.repository_config.url
   end
+
+  def restrictions
+    document.collection.fetch('accessrestrict_html_tesm', ['']).first.html_safe # rubocop:disable Rails/OutputSafety
+  end
+
+  def render?
```

Comment:
Using this `render?` method is clean and good, but it makes reading `document_component.html.erb` confusing:
https://github.com/sul-dlss/stanford-arclight/blob/ad3428e7de3a65e1238657b3c99829e0ebdf0c30/app/components/arclight/document_component.html.erb#L29

That template has an `<% if document.collection? %>` within it, but `<%= render UsingTheseMaterialsComponent.new(document:) %>` stands outside of it. So the logic within that template about what to render only on a collection page is split between multiple places, making it harder to track down.

Reply:
We can create a cleanup ticket in AL Core for the document_component file.